R/google_analytics.R

```diff
@@ -31,12 +44,31 @@ getGoogleAnalytics <- function(tableId, lastNDays = 30, dimensions, metrics, tok
   }
   token <- getGoogleTokenForAnalytics(tokenFileId)
 
-  start_date <- as.character(lubridate::today() - lubridate::days(lastNDays))
-  #end_date <- as.character(lubridate::today() - lubridate::days(1))
-  end_date <- as.character(lubridate::today())
+  if(dateRangeType == "lastNDays") {
+    if(is.null(lastN)) {
+      # For backward compatibility for Exploratory Desktop older than verssion 5.4.1
```

Comment:
Typo: verssion

Reply:
Thank you for catching it. Fixed the typo.